tests/test_init.py

```diff
@@ -6,7 +6,7 @@
 from unittest.mock import patch
 import pytest
 
-import tomli
+from flit_core._vendor import tomli
```

Comment:
Let's revert this and ensure `tomli` is installed for tests in Flit (not `flit_core`).